File: onde_esta_o_navio.cpp
#include <iostream>
using namespace std;

int main(){
    int X,Y;
    cin >> X >> Y;
    if(0 < X && X < 100 && 0 < Y && Y < 100){
        if(X > 70 || Y > 70){
            cout << "Coordenada valida e o navio esta longe";
        }else{
            cout << "Coordenada valida e o navio esta perto";
        }
    }else{
        cout << "Coordenada invalida";
    }
}

File: onde_esta_o_navio.py
X,Y = map(int,input().split())
if 0 < X < 100 and 0 < Y < 100:
    if X <71 or Y < 71:
        print("Coordenada valida e o navio esta perto")
    else:
        print("Coordenada valida e o navio esta longe")
else:
    print("Coordenada invalida")